### DISCUSSION TITLE
Multiple versions of doc

### DISCUSSION THREAD

**Discussion by galleon** [vuepress/core#466] [Q&A] [answered]
Hi All

First I need to mention that I am new to VuePress (and Vue) :-) 
I need to present multiple versions of the documentation of an open-source library. I know this topic has been addressed a number of time with VuePress 1.x but I thought I would ask the community how to deal with that with the new version.

As I am using GH Pages I thought I could have a GH Action that would change the `base` config parameter on the fly as a new release is generated and push that in a specific directory in the `gh_pages` branch. But would that be sufficient - or is there a better way to do it?

Thanks!

**Comment by bidoubiwa** [comment]
I'm really curious about this as well. Or alternatively, contribute to add that feature. Is this a feature that we want on vuepress-next or is this something that should be implemented as a plugin?

**Comment by Mister-Hope** [comment, marked as answer]
> Is this a feature that we want on vuepress-next or is this something that should be implemented as a plugin?

First, IMO this cannot be implemented as a plugin (or really hard and hacky)

It depends on your thoughs, do you preffer to deploy each version of your docs as a standalone vuepress project, or you preffer to place them in subfolders of a single project which will be later rendered under different path with version prefix.

- If you prefer the first one, the vuepress already finish it's job, it builds a docs site for each version of your docs.
- If you prefer the second one, a plugin could be made to autocomplete some jobs for you or make some optimization.
